lucene/packaging/build.gradle

```diff
@@ -32,7 +32,9 @@ def includeInBinaries = project(":lucene").subprojects.findAll {subproject ->
         ":lucene:packaging",
         ":lucene:documentation",
         // Exclude parent container project of analysis modules (no artifacts).
-        ":lucene:analysis"
+        ":lucene:analysis",
+        // Exclude native module, which requires manual copying and enabling
+        ":lucene:native"
```

Comment:
This reference needs an updated path - it'll probably fail when building the whole project.

Reply:
It did fail, I've updated the reference here and confirmed it's able to build from directory root.